source/includes/kubernetes_extension/_k8_services.md

```diff
@@ -0,0 +1,120 @@
+### Services
+
+<!-------------------- LIST SERVICES -------------------->
+
+#### List services
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/services"
```

Comment:
This document is missing the cluster ID. Please see the other includes for examples. The API won't work without it. This doesn't apply to the standalone plugin.

Reply:
Good catch, thank you!

Reply:
Done! 6e9f126e5c49c06174281e6419a386321fc7c2cf